main.py

```diff
@@ -0,0 +1,32 @@
+from app.animal import Animal, Carnivore, Herbivore
+
+
+if __name__ == "__main__":
+    lion = Carnivore("Simba")
+    print(len(Animal.alive) == 1)
+    print(isinstance(Animal.alive[0], Carnivore) is True)
+
+    rabbit = Herbivore("Susan")
+    rabbit.hide()
+    print(rabbit.hidden is True)
+
+    lion = Carnivore("Lion King")
+    rabbit = Herbivore("Susan")
+    print(rabbit.health == 100)
+    lion.bite(rabbit)
+    print(rabbit.health == 50)  # bited
+
+    rabbit.hide()
+    lion.bite(rabbit)
+    print(rabbit.health == 50)  # lion cannot bite hidden rabbit
```

Comment:
Code Efficiency: Use `assert` statement to check the condition. For example, `assert rabbit.health == 50`.